src/fparser/two/tests/test_symbol_table.py

```diff
@@ -279,6 +279,34 @@ def test_module_use_with_rename(f2003_parser):
     assert table.lookup("other").primitive_type == "logical"
 
 
+def test_wildcard_module_search(f2003_parser):
+    """Test the wildcard_imports method of the SymbolTable."""
```

Comment:
Good stuff, but does it work in other situations.

Reply:
Extended.

Reply:
Thanks.